xga/sources/base.py

```diff
@@ -400,7 +400,8 @@ def __init__(self, ra: float, dec: float, redshift: float = None, name: str = No
         # The nh_lookup function returns average and weighted average values, so just take the first. If this is a
         #  BaseSource and part of a sample then we're going to avoid the call to nh_lookup, for efficiency
         if in_sample and type(self) == BaseSource:
-            self._nH = Quantity(np.NaN, 'cm^-2')
+            # FIXME: Compatability -- np.NaN deprecated and replaced by np.nan in numpy >= 2
+            self._nH = Quantity(np.nan, 'cm^-2')
```

Comment:
Didn't know about this deprecation, cheers

Reply:
No worries, I think it's just a warning at the moment but worth dealing with head-on.